apps/experiments/models.py

```diff
@@ -447,6 +435,28 @@ class Meta:
         )
 
 
+class ExperimentVersion(models.Model):
+    experiment = models.ForeignKey(Experiment, on_delete=models.CASCADE, related_name="versions")
+    version_data = JSONField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    is_default = models.BooleanField(default=False)
```

Comment:
Wondering about indexing on the `is_default` field. I can imagine us using this a lot

Reply:
It would need to be a partial index. In Django you can do that by creating an Index with a condition: https://docs.djangoproject.com/en/5.0/ref/models/indexes/#condition